glossary/comet.md

```diff
@@ -6,9 +6,20 @@ template = "doc.html"
 category = "arvo"
 +++
 
-A **comet** is a kind of [ship](../ship) on the [Arvo](../arvo) network. They are disposable identities that will not be initially trusted by others on the network. One use might be to run bots as comets.
+A **comet** is a kind of [ship](../ship) on the [Arvo](../arvo) network. They
+are free identities for which we encourage new users to boot to try things out
+before deciding to purchase a [planet](..planet).
 
-Comets have no presence on [Azimuth](../azimuth) and thus have no associated Urbit identities.
+Comets have nearly all the same capabilities as a planet. The exceptions are:
+ - They may not spawn [moons](../moon).
+ - They may not [change their networking keys](../bridge).
```

Comment:
"can not" they aren't prevented from doing this by some authority, which this sounds like they are. Should explicitly mention here that it's because they are not on chain so there is no way to broadcast a trustable change in keys.